### DISCUSSION TITLE
Cannot open Lucene index using Luke.

### DISCUSSION THREAD

**Discussion by mrpavlitzky** [OrchardCMS/OrchardCore#12099] [Q&A] [answered]
Hi, I'm trying to work out a full-text search in my application and I have trouble using Luke. I set up Lucene module and search index and I even managed to find the blog posts by title using /search URL. But when trying to access the index using Luke it tells me there's no valid directory at the location (/App_Data/Sites/Default/Lucene/PostsIndex). When I open the directory I can see the following files:

- _0.cfe
- _0.cfs
- _0.si
- segments.gen
- segments_2

Any suggestions/help would be greatly appreciated.

Cheers

**Comment by mrpavlitzky** [comment, marked as answer]
Solved: 

Luke 5.5.0 works as intended

**Comment by Skrypt** [comment]
I've been successfully using Luke 4.10.4 recently.
